tests/test_bgp_neighbor_af.rb

```diff
@@ -226,12 +233,14 @@ def test_tri_states
     @@matrix.values.each do |af_args|
       af, dbg = clean_af(af_args)
 
-      %w(additional_paths_receive additional_paths_send).each do |k|
-        [:enable, :disable, :inherit, 'enable', 'disable', 'inherit',
-         af.send("default_#{k}")
-        ].each do |val|
-          af.send("#{k}=", val)
-          assert_equal(val.to_sym, af.send(k), "#{dbg} Error: #{k}")
+      unless dbg.include?('l2vpn/evpn')
```

Comment:
I can not use next if here, because I need to test the soft_reconfiguration_in after that.